recipe/meta.yaml

```diff
@@ -8,16 +8,20 @@ package:
 source:
   url: https://github.com/stack-of-tasks/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
   sha256: 958b6c971d061341eadacaa293c5cf34f5f94ecc1a3c04383d366a5a9b198bf7
+  patches:
+    - windows.patch
 
 build:
   number: 0
-  skip: true  # [win]
+  skip: true  # [win and vc<14]
 
 requirements:
   build:
-    - {{ compiler('cxx') }}
+    - {{ compiler('cxx') }}  # [not win]
+    - vs2017_win-64  # [win64]
```

Comment:
Shouldn’t the compiler('cxx') stuff also work for win?

Reply:
I asked on the gitter channel and apparently it doesn't work if you want vs2017 (as opposed to 2014) which I guess is intended here :)

Reply:
Thanks for the confirmation. I will keep it as is. Btw, I learned this trick from `qt-feedstock`. :)